frontend-dev/src/pages/AuthResponse.jsx

```diff
@@ -1,17 +1,17 @@
 import { useEffect } from 'react'
 import { useRecoilState } from 'recoil'
-import { grantTokenAtom } from '../GlobalStates'
+import { authInfoAtom } from '../GlobalStates'
 
 // popup window: render when redirected from oauth to bit-integration with code
 export default function AuthResponse() {
-  const [, setGrantToken] = useRecoilState(grantTokenAtom)
+  const [, setAuthInfo] = useRecoilState(authInfoAtom)
 
   useEffect(() => {
     const urlParams = new URLSearchParams(window.location.hash)
     const code = urlParams.get('code')
 
     if (code) {
-      setGrantToken(code)
+      setAuthInfo(code)
```

Comment:
authInfo is an object but you are putting a string in this state it should be line something 
```
{
    code: 'your_code'
}
```